kore-rpc-types/src/Kore/JsonRpc/Types.hs

```diff
@@ -157,7 +157,7 @@ data Condition = Condition
 
 data ImpliesResult = ImpliesResult
     { implication :: KoreJson
-    , satisfiable :: Bool
+    , valid :: Bool
     , condition :: Maybe Condition
     , logs :: Maybe [LogEntry]
     }
```

Comment:
Maybe better to [update the renamed field in the JSON pretty-printing configuration](https://github.com/runtimeverification/haskell-backend/blob/bf0ea9ff15100adeab0bfdd787ccce7af0c191fe/kore-rpc-types/src/Kore/JsonRpc/Types.hs#L286) (which contains a field order that should make sense to most readers).